docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -166,6 +169,8 @@ Before you begin, [create a cluster](#create-swarms). Note the its **Name** and
    ...
    ```
 
+   **Tip**: If you need to get the **Name** and **ID** of your cluster, run `bx d4ic list --sl-user SOFTLAYER_USERNAME --sl-api-key SOFTLAYER_API_KEY`.
+
```

Comment:
If you really want that to show up as a tip, you need the following on this line and then add a newline:

```liquid
{:.tip}
```

That adds `class="tip"` to the note.